rules/code-quality/tests/Rector/Identical/GetClassToInstanceOfRector/Fixture/static_should_not_be_namespaced.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\CodeQuality\Tests\Rector\Identical\GetClassToInstanceOfRector\Fixture;
+
+class StaticShouldNotBeNamespaced {
+    public function test() {
+        if (\get_class($other) !== static::class) {}
+    }
```

Comment:
it may need flip check? `if (static::class !== \get_class($other)`

Reply:
It would be convenient, but not necessary